### PR TITLE
Add check_permissions to check_mediafile_id

### PR DESCRIPTION
**Pull request by reiterl**
Add some tests.

### CODE REVIEW

File: openslides_backend/presenter/check_mediafile_id.py [hunk 1]
```diff
@@ -37,7 +48,99 @@ def get_result(self) -> Any:
...
+        meeting = self.datastore.get(
+            FullQualifiedId(Collection("meeting"), meeting_id),
+            ["admin_group_id", "enable_anonymous", "user_ids", "committee_id"],
+        )
+        groups_field = f"group_${meeting_id}_ids"
+        user = self.datastore.get(
+            FullQualifiedId(Collection("user"), self.user_id), [groups_field]
+        )
+        if meeting.get("admin_group_id") in user.get(groups_field, []):
+            return
```
Comment (jsangmeister) [inline openslides_backend/presenter/check_mediafile_id.py:86]: Please move this to a helper function `is_admin` (together with the above check for superadmin) and use that here and [in this meeting.update test](https://github.com/OpenSlides/openslides-backend/blob/6db4f187b50e07900569a287b4ae541a0f6cf12c/openslides_backend/action/actions/meeting/update.py#L228) (and in any other places if you find them, on a quick search I did not find more)

File: openslides_backend/presenter/check_mediafile_id.py [hunk 2]
```diff
@@ -37,7 +48,99 @@ def get_result(self) -> Any:
...
+            user_groups = set(user.get(f"group_${meeting_id}_ids", []))
+            if inherited_access_group_ids & user_groups:
+                return
+        raise PermissionDenied("You are not allowed to see this mediafile.")
+
+    def check_can_see_meeting(self, meeting: Dict[str, Any]) -> bool:
+        """needs meeting to include enable_anonymous, user_ids, committee_id."""
+        if has_organization_management_level(
+            self.datastore, self.user_id, OrganizationManagementLevel.SUPERADMIN
+        ):
```
Comment (jsangmeister) [inline openslides_backend/presenter/check_mediafile_id.py:132]: You can remove this check here since it was already checked earlier in the other function

**Review by jsangmeister — CHANGES_REQUESTED** [review]
One test is missing for the last point: The user has mediafile.can_see and the user has groups in common with meeting/inherited_access_group_ids.

**Comment by reiterl** [comment]
Rebased the branch to solve the conflict.

File: openslides_backend/presenter/check_mediafile_id.py (hunk 1 above)
Comment (reiterl) [reply openslides_backend/presenter/check_mediafile_id.py]: Moved a is_admin function into the permission_helpers and use it at meeting.update.

File: openslides_backend/presenter/check_mediafile_id.py (hunk 2 above)
Comment (reiterl) [reply openslides_backend/presenter/check_mediafile_id.py]: Removed.

**Comment by reiterl** [comment]
Add a test for the inherited_access_group.

**Review by jsangmeister — APPROVED** [review]

**Pull request merged**